src/main/java/com/onaio/steps/handler/actions/NotReachableHandler.java

```diff
@@ -29,10 +29,10 @@
 
 public class NotReachableHandler implements IMenuHandler,IMenuPreparer {
 
-    private IDoNotTakeSurveyStrategy refusedSurveyStrategy;
+    private final IDoNotTakeSurveyStrategy refusedSurveyStrategy;
```

Comment:
Why make these 3 final? I believe we can use inheritance if we want to override these values in the child class instead of redeclaring them

Reply:
these three fields were private previously and cannot be inherited. However, I made these final because all these are initialized on the object creation state. i.e in the constructor.